docs/articles/retrieval_augmented_generation_eval_qdrant_arize.md

```diff
@@ -0,0 +1,526 @@
+    provide_explanation=True,
+)
+relevance_eval_df_hybrid = run_evals(
+    dataframe=retrieved_documents_df_hybrid,
+    evaluators=[relevance_evaluator],
+    provide_explanation=True,
+)[0]
+
+px.Client().log_evaluations(
+    SpanEvaluations(eval_name="Hallucination", dataframe=hallucination_eval_df_hybrid),
+    SpanEvaluations(eval_name="QA Correctness", dataframe=qa_correctness_eval_df_hybrid),
+    project_name=HYBRID_RAG_PROJECT,
+)
+px.Client().log_evaluations(DocumentEvaluations(eval_name="Relevance", dataframe=relevance_eval_df_hybrid),
+                            project_name=HYBRID_RAG_PROJECT)
+```
+
+You will notice your project will now have 2 workspaces as below :
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_project_workspace_with_2_projects.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_project_workspace_with_2_projects.png)
+
+Lets dig into our hybrid-rag workspace.
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_hybrid_rag_eval.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/arize_hybrid_rag_eval.png)
+
+Let’s take a quick look at the stats and we notice :
+
+`Hallucination` is 0.00 and `QA Correctness` is 1.00 .
+
+Also , notice the question `What is the purpose of ef_construct in HNSW ?` which earlier showed Hallucinated, now shows as :
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/hybrid_rag_trace_detail.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/hybrid_rag_trace_detail.png)
+
+and in the `Evaluation` tab :
+
+![../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/hybrid_rag_trace_eval_tab.png](../assets/use_cases/retrieval_augmented_generation_eval_qdrant_arize/hybrid_rag_trace_eval_tab.png)
+
+seem to be already fixed.
+
+**Concluding thoughts**
+
+In the [previous article]([https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas](https://superlinked.com/vectorhub/articles/retrieval-augmented-generation-eval-qdrant-ragas)) we experimented improving RAG Evaluation score by experimenting with different *Retrieval Window sizes* while in this article we covered *Multivector / Hybrid vector* approach to improve and fix our RAG pipeline.
+
+It is suggested to correlate these experiments with your use case.
+
+There are other ways of improving your RAG pipeline ,  such as :
+
+- Experimenting with *different chunk sizes and chunk overlap settings* - based on the nature of queries and response expectations.
+- Experimenting with *different embedding model and LLMs* - based on domain understanding and response capabilities as required
+- Experimenting with *rerankers* - based of preset or custom relevance criterion
+- Experimenting with *query and response enrichment* - based on level of enrichment required before and after retrieval state to help LLM generate the desired response.
```

Comment:
Please add a point about combining information from different sources, like metadata of text + text itself